packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -20,6 +20,7 @@ export namespace CommonMenus {
     export const FILE_NEW = [...FILE, '1_new'];
     export const FILE_OPEN = [...FILE, '2_open'];
     export const FILE_SAVE = [...FILE, '3_save'];
+    export const WORKSPACE_CLOSE = [...FILE, '4_close'];
```

Comment:
It is a command group, should be `FILE_CLOSE`, core extension is not aware about workspaces